README.md.old

```diff
@@ -41,7 +55,58 @@ Checkout our [examples](./examples/README.md) directory for complete examples of
 Here is a quickstart you can use in your own project:
 
 ```typescript
-{{ usageExampleCode }}
```

Comment:
Did you want to delete this file?

Reply:
not yet.  there's a bunch of content in here that needs to get migrated over to dev docs so was just leaving this as a reminder